src/components/mentor/mentorshipsList.tsx

```diff
@@ -1,96 +1,165 @@
   }
 
   public render() {
+    const { activeMentorships, pendingMentorships, selectedTab } = this.state;
+    const { classes } = this.props;
+
     return (
       <MentorshipsListWrapper>
-        {this.state.mentorships.length > 0 ? (
+        <div className={classes.root}>
+          <AppBar position="static">
+            <Tabs value={selectedTab} onChange={this.handleChange}>
+              <Tab label="active" href="#active" />
+              <Tab label="pending" href="#pending" />
+            </Tabs>
+          </AppBar>
+          {selectedTab === 0 && (
+            <TabContainer>
+              {this.displayTabContainerMentorships(
+                activeMentorships,
+                "You don't have any active mentorships. Get started by sending a mentor request."
+              )}
+            </TabContainer>
+          )}
+          {selectedTab === 1 && (
+            <TabContainer>
+              {this.displayTabContainerMentorships(
+                pendingMentorships,
+                "You don't have any pending mentorships. Get started by sending a mentor request."
+              )}
+            </TabContainer>
+          )}
+        </div>
+      </MentorshipsListWrapper>
+    );
+  }
+
+  private displayTabContainerMentorships(mentorships: IMentorship[], noContentMessage: string): JSX.Element {
+    if (mentorships.length > 0) {
+      return (
+        <TabContainer>
           <List>
-            {this.state.mentorships.map(mentorship => (
+            {mentorships.map(mentorship => (
               <ListItem key={mentorship.Id} dense={true} button={true}>
                 <Avatar
-                  alt={`${mentorship.User!.FirstName} ${mentorship.User!.LastName}`}
-                  src={
-                    mentorship.User!.PictureUrl === null ? Const.DefaultProfileImagePath : mentorship.User!.PictureUrl
-                  }
+                  alt={`${mentorship.Mentor!.FirstName} ${mentorship.Mentor!.LastName}`}
+                  src={mentorship.Mentor!.PictureUrl}
                 />
                 <ListItemText
-                  primary={`${mentorship.User!.FirstName} ${mentorship.User!.LastName}`}
-                  secondary={mentorship.User!.Objective}
+                  primary={`${mentorship.Mentor!.FirstName} ${mentorship.Mentor!.LastName}`}
+                  secondary={`${mentorship.Mentor!.CompanyTitle} with ${mentorship.Mentor!.CompanyName}`}
                 />
               </ListItem>
             ))}
           </List>
-        ) : (
-          <NoContent message={"You don't have any active mentorships. Wait for requests."} />
-        )}
-      </MentorshipsListWrapper>
-    );
+        </TabContainer>
+      );
+    } else {
+      return (
+        <TabContainer>
+          <NoContent message={noContentMessage} />
+        </TabContainer>
+      );
+    }
+  }
+
+  private handleChange(e: React.FormEvent<HTMLSelectElement>, selectedTab: number) {
```

Comment:
If you use a fat arrow syntax here, the method binding inside the constructor can be removed.

```suggestion
  private handleChange = (e: React.FormEvent<HTMLSelectElement>, selectedTab: number) => {
```